# SPCE5025

## Spring 2025

## Exam 3

**Instructions:** This exam is open-book. You may use other resources as you need them, so long as you cite your sources if you use them in a response. You may not collaborate with other people on this exam.

**Exam Contents:** The problems are similar to what you have done in the homework.

**Due Date:** Saturday, 11 April 2025 11:59 p.m.

**Grading:** This exam has 3 problems and is worth a total of 200 points. Grades will be assigned as a percentage from 0-100%.

**If you have questions:** If you have questions about the content or meaning of the exam problems, you may e-mail me. As appropriate I will post the question and answers on Canvas for everyone’s benefit.

I will also be available to discuss questions on the Canvas conference at the regularly-scheduled times:

**When:**

- Tuesday, 07 April, 7:00-9:00 pm, MDT
- Thursday, 09 April, 7:00-9:00 pm, MDT

**Spring Break:** We will observe Spring Break during the week of 23-27 March. There will be no office hours during that week, but I will monitor e-mail and post announcements as appropriate for questions and answers received.

## Ground Site visibility and Radio Frequency Interference

Two very common computations in satellite operations are to determine:

1. When a satellite is within view of a ground site
2. When the angular separation between two satellites, as seen from a ground site, is within some threshold value.

For this exam you will perform these computations using the techniques from the Week 9 and 10 notes.

**Reference:** True of Date (TOD)  
**Epoch:** 20-Apr-2012 / 00:00:00.0

### Vehicle 1

| Parameter | Value                  |
|-----------|------------------------|
| X         | -12211757.775959 m     |
| Y         | 9429062.442371 m       |
| Z         | -21428647.221756 m     |
| XD        | -2036.625445 m/sec     |
| YD        | -3312.346224 m/sec     |
| ZD        | -247.989882 m/sec      |
| A         | 26555009.241483 m      |
| E         | 0.011644               |
| I         | 54.559609 deg          |
| RAAN      | 241.016005 deg         |
| WP        | 326.488375 deg         |
| NU        | 298.447619 deg         |
| MA        | 299.615951 deg         |
| LATD      | -54.172764 deg         |
| LON       | 293.841985 deg         |
| ALTD      | 20040.916981 km        |

### Vehicle 2

| Parameter | Value                  |
|-----------|------------------------|
| X         | -15365447.628128 m     |
| Y         | 13189062.630639 m      |
| Z         | -17181708.494580 m     |
| XD        | -3150.030141 m/sec     |
| YD        | -1134.516168 m/sec     |
| ZD        | 1949.677579 m/sec      |
| A         | 26559659.083624 m      |
| E         | 0.000597               |
| I         | 55.025775 deg          |
| RAAN      | 175.768144 deg         |
| WP        | 27.213768 deg          |
| NU        | 280.640888 deg         |
| MA        | 280.708129 deg         |
| LATD      | -40.278601 deg         |
| LON       | 290.873364 deg         |
| ALTD      | 20187.562120 km        |

### Ground Site

| Site  | Latitude (deg) | Longitude (deg) | Altitude (m) |
|-------|----------------|-----------------|--------------|
| POGO  | 76.515         | 291.400         | 132.000      |

## Instructions, Notes, and Reminders

a. Note that the input reference frame for the propagations is True of Date (TOD), and the output frame will also be TOD.

b. The equations used to create a ground site vector from latitude, longitude, and altitude, return the vector in the Earth Centered, Earth Fixed (ECEF) reference frame.

c. Azimuth and elevation from a ground site to a satellite are computed in the Topocentric reference frame.

d. The TOD→ECEF transformation is a Z-rotation with an angular magnitude corresponding to the Greenwich Hour Angle (GHA).

e. For the final two problems it will probably be necessary to interpolate between time points of the propagation to find times where the satellite(s) cross a particular angular threshold. You can use linear interpolation to do this. (Note that you will get more accurate interpolation results if your trajectory results are available at a smaller step size.)

## 1. Orbit Propagations (50 points)

Using a stepsize of 200 seconds, propagate each vector for 12 hours (216 steps), using central body gravity only.

Present your results in tabular form:

| Time from Epoch (sec) | X (m) | Y (m) | Z (m) | XD (m/sec) | YD (m/sec) | ZD (m/sec) |
|-----------------------|-------|-------|-------|------------|------------|------------|
|                       |       |       |       |            |            |            |

**Note:** For computational convenience, you may wish to combine the computation and output for both satellites, and for Problems 2 and 3 in the same code and output file as for Problem 1.



In [ ]:
from standards import *
import copy

### Ground sensor
SEN_LAT = radians(76.515) #rad
SEN_LON = radians(291.400) #rad
SEN_ALTD = 132.000 #m
R_ECEF_TOPOCENTRIC = np.array([
    [-sin(SEN_LON), cos(SEN_LON), 0],
    [-sin(SEN_LAT)*cos(SEN_LON), -sin(SEN_LAT)*sin(SEN_LON), cos(SEN_LAT)],
    [ cos(SEN_LAT)*cos(SEN_LON), cos(SEN_LAT)*sin(SEN_LON), sin(SEN_LAT)]
])

### Vehicle 1
V1_X    = -12211757.775959 #m
V1_Y    = 9429062.442371 #m  
V1_Z    = -21428647.221756 #m
V1_XD   = -2036.625445 #m/sec
V1_YD   = -3312.346224 #m/sec
V1_ZD   = -247.989882 #m/sec 
V1_A    = 26555009.241483 #m 
V1_E    = 0.011644          
V1_I    = 54.559609 #deg     
V1_RAAN = 241.016005 #deg    
V1_WP   = 326.488375 #deg    
V1_NU   = 298.447619 #deg    
V1_MA   = 299.615951 #deg    
V1_LATD = -54.172764 #deg    
V1_LON  = 293.841985 #deg    
V1_ALTD = 20040.916981 #km 
V1_Pos = Vector3(V1_X, V1_Y, V1_Z)
V1_Vel = Vector3(V1_XD, V1_YD, V1_ZD)

### Vehicle 2
V2_X    = -15365447.628128 #m
V2_Y    = 13189062.630639 #m 
V2_Z    = -17181708.494580 #m
V2_XD   = -3150.030141 #m/sec
V2_YD   = -1134.516168 #m/sec
V2_ZD   = 1949.677579 #m/sec 
V2_A    = 26559659.083624 #m 
V2_E    = 0.000597          
V2_I    = 55.025775 #deg     
V2_RAAN = 175.768144 #deg    
V2_WP   = 27.213768 #deg     
V2_NU   = 280.640888 #deg    
V2_MA   = 280.708129 #deg    
V2_LATD = -40.278601 #deg    
V2_LON  = 290.873364 #deg    
V2_ALTD = 20187.562120 #km
V2_Pos = Vector3(V2_X, V2_Y, V2_Z)
V2_Vel = Vector3(V2_XD, V2_YD, V2_ZD)
Step_Size = 200
Step_Num = 216
start_time = datetime(year=2012, month=4, day=20,hour=0, minute=0, second=0)
Earth_rotation = 72.921151467e-6 #rad/sec
Earth_gravitational_parameter = 3.986004418e14 #m^3/s^2
Earth_radius = 6378137 #m
f = 1/298.257223563
Earth_Eccentricity = sqrt(2*f-f**2)


class Step:
    def __init__(self, T, V1_RK_X, V1_RK_Y, V1_RK_Z, V1_RK_XD, V1_RK_YD, V1_RK_ZD, V1_LAT, V1_LON, V1_ALTD, V1_GAH,
                 V2_RK_X, V2_RK_Y, V2_RK_Z, V2_RK_XD, V2_RK_YD, V2_RK_ZD, V2_LAT, V2_LON, V2_ALTD, V2_GAH):
        self.T = T
        self.V1_RK_X = V1_RK_X
        self.V1_RK_Y = V1_RK_Y
        self.V1_RK_Z = V1_RK_Z
        self.V1_RK_XD = V1_RK_XD
        self.V1_RK_YD = V1_RK_YD
        self.V1_RK_ZD = V1_RK_ZD
        self.V1_LAT = V1_LAT
        self.V1_LON = V1_LON
        self.V1_ALTD = V1_ALTD
        self.V1_GAH = V1_GAH
        self.V2_RK_X = V2_RK_X
        self.V2_RK_Y = V2_RK_Y
        self.V2_RK_Z = V2_RK_Z
        self.V2_RK_XD = V2_RK_XD
        self.V2_RK_YD = V2_RK_YD
        self.V2_RK_ZD = V2_RK_ZD
        self.V2_LAT = V2_LAT
        self.V2_LON = V2_LON
        self.V2_ALTD = V2_ALTD
        self.V2_GAH = V2_GAH

steps_central_body = []
V1_y_0 = Vector6(V1_Pos.x, V1_Pos.y, V1_Pos.z, V1_Vel.x, V1_Vel.y, V1_Vel.z)
V2_y_0 = Vector6(V2_Pos.x, V2_Pos.y, V2_Pos.z, V2_Vel.x, V2_Vel.y, V2_Vel.z)
h = Step_Size
w_ = Vector3(0, 0, 72.921151467e-6)
V1_v_r_ = V1_Vel - (w_.cross(V1_Pos))
V2_v_r_ = V2_Vel - (w_.cross(V2_Pos))
V1_temp_pos = copy.deepcopy(V1_Pos)
V2_temp_pos = copy.deepcopy(V2_Pos)
V1_temp_vel = copy.deepcopy(V1_Vel)
V2_temp_vel = copy.deepcopy(V2_Vel)
timestamp = start_time
V1_gah, V1_pos_ECEF_np = rotate_TOD_to_ECEF(start_time, V1_Pos, Earth_rotation)
V2_gah, V2_pos_ECEF_np = rotate_TOD_to_ECEF(start_time, V2_Pos, Earth_rotation)

steps_central_body.append(Step(0, V1_y_0.x, V1_y_0.y, V1_y_0.z, V1_temp_vel.x, V1_temp_vel.y, V1_temp_vel.z, V1_LATD, V1_LON, V1_ALTD, V1_gah,
V2_y_0.x, V2_y_0.y, V2_y_0.z, V2_temp_vel.x, V2_temp_vel.y, V2_temp_vel.z, V2_LATD, V2_LON, V2_ALTD, V2_gah))

for i in range(Step_Num):
    t = h*(i+1)
    timestamp = timestamp + timedelta(seconds=h)
    V1_step = compute_rk_2_body_step(0, Earth_gravitational_parameter, V1_temp_pos, V1_temp_vel, h, V1_y_0, timestamp, 0, 0, 0)
    V2_step = compute_rk_2_body_step(0, Earth_gravitational_parameter, V2_temp_pos, V2_temp_vel, h, V2_y_0, timestamp, 0, 0, 0)
    V1_gah, V1_pos_ECEF_np = rotate_TOD_to_ECEF(timestamp, Vector3(V1_step.x, V1_step.y, V1_step.z), Earth_rotation)
    V2_gah, V2_pos_ECEF_np = rotate_TOD_to_ECEF(timestamp, Vector3(V2_step.x, V2_step.y, V2_step.z), Earth_rotation)
    SEN_gah, SEN_pos_ECEF_np = rotate_TOD_to_ECEF(timestamp, Vector3(V2_step.x, V2_step.y, V2_step.z), Earth_rotation)
    V1_pos_ECEF = Vector3(V1_pos_ECEF_np[0][0], V1_pos_ECEF_np[1][0], V1_pos_ECEF_np[2][0])
    V2_pos_ECEF = Vector3(V2_pos_ECEF_np[0][0], V2_pos_ECEF_np[1][0], V2_pos_ECEF_np[2][0])
    V1_lat, V1_lon, V1_height = calculate_lat_lon_h(Earth_Eccentricity, V1_pos_ECEF, Earth_radius)
    V2_lat, V2_lon, V2_height = calculate_lat_lon_h(Earth_Eccentricity, V2_pos_ECEF, Earth_radius)


    print(f"ECEF to Topocentric rotation matrix: \n{R_ECEF_TOPOCENTRIC}\n")
    V1_TOCOCENTRIC_Pos_sat = R_ECEF_TOPOCENTRIC @ V1_pos_ECEF_np
    V2_TOCOCENTRIC_Pos_sat = R_ECEF_TOPOCENTRIC @ V2_pos_ECEF_np

    TOPOCENTRIC_Pos_sen = R_ECEF_TOPOCENTRIC @ Pos_2_sensor_.get_np_vector()
    sen_to_sat_ = TOCOCENTRIC_Pos_sat_ - TOPOCENTRIC_Pos_sen_
    print(f"Relative Distance Between Sat and Sensor in Topocentric: \n{sen_to_sat_}\n")
    Az = atan2(sen_to_sat_[0][0], sen_to_sat_[1][0])
    El = atan(sen_to_sat_[2][0]/sqrt(sen_to_sat_[0][0]**2 + sen_to_sat_[1][0]**2))
    print(f"Azimuth: {degrees(Az)}")
    print(f"Elevation: {degrees(El)}")

    steps_central_body.append(Step(t, V1_step.x, V1_step.y, V1_step.z, V1_step.xd, V1_step.yd, V1_step.zd, V1_lat, V1_lon, V1_height, V1_gah,
        V2_step.x, V2_step.y, V2_step.z, V2_step.xd, V2_step.yd, V2_step.zd, V2_lat, V2_lon, V2_height, V2_gah))
    V1_y_0 = V1_step
    V2_y_0 = V2_step
    V1_temp_pos = Vector3(V1_step.x, V1_step.y, V1_step.z)
    V1_temp_vel = Vector3(V1_step.xd, V1_step.yd, V1_step.zd)
    V2_temp_pos = Vector3(V2_step.x, V2_step.y, V2_step.z)
    V2_temp_vel = Vector3(V2_step.xd, V2_step.yd, V2_step.zd)

from IPython.display import Markdown, display

header = "| Step | Time (s) | V1_X (m) | V1_Y (m) | V1_Z (m) | V1_XD (m/s) | V1_YD (m/s) | V1_ZD (m/s) | V1_GAH (deg) | V1_LON (deg) | V1_LAT (deg) | V1_ALT (m) | V2_X (m) | V2_Y (m) | V2_Z (m) | V2_XD (m/s) | V2_YD (m/s) | V2_ZD (m/s) | V2_GAH (deg) | V2_LON (deg) | V2_LAT (deg) | V2_ALT (m) |"
separator = "|------|----------|----------|----------|----------|--------------|--------------|--------------|-----------|-----------|---------|---------|----------|----------|----------|--------------|--------------|--------------|-----------|-----------|---------|---------|"

rows = []
for i, s in enumerate(steps_central_body):
    row = (
        f"| {i} | {s.T} | {s.V1_RK_X:.2f} | {s.V1_RK_Y:.2f} | {s.V1_RK_Z:.2f} | {s.V1_RK_XD:.2f} | {s.V1_RK_YD:.2f} | {s.V1_RK_ZD:.2f} | {s.V1_GAH:.4f} | {s.V1_LON:.4f} | {s.V1_LAT:.4f} | {s.V1_ALTD:.2f}"
        f"| {s.V2_RK_X:.2f} | {s.V2_RK_Y:.2f} | {s.V2_RK_Z:.2f} | {s.V2_RK_XD:.2f} | {s.V2_RK_YD:.2f} | {s.V2_RK_ZD:.2f} | {s.V2_GAH:.4f} | {s.V2_LON:.4f} | {s.V2_LAT:.4f} | {s.V2_ALTD:.2f} |"
    )
    rows.append(row)

table_md = "\n".join([header, separator] + rows)

display(Markdown(f"### RK4 Integration Steps (State + Geodetic)\n\n{table_md}"))


### RK4 Integration Steps (State + Geodetic)

| Step | Time (s) | V1_X (m) | V1_Y (m) | V1_Z (m) | V1_XD (m/s) | V1_YD (m/s) | V1_ZD (m/s) | V1_GAH (deg) | V1_LON (deg) | V1_LAT (deg) | V1_ALT (m) | V2_X (m) | V2_Y (m) | V2_Z (m) | V2_XD (m/s) | V2_YD (m/s) | V2_ZD (m/s) | V2_GAH (deg) | V2_LON (deg) | V2_LAT (deg) | V2_ALT (m) |
|------|----------|----------|----------|----------|--------------|--------------|--------------|-----------|-----------|---------|---------|----------|----------|----------|--------------|--------------|--------------|-----------|-----------|---------|---------|
| 0 | 0 | -12211757.78 | 9429062.44 | -21428647.22 | -2036.63 | -3312.35 | -247.99 | 79.0369 | 293.8420 | -54.1728 | 20040.92| -15365447.63 | 13189062.63 | -17181708.49 | -3150.03 | -1134.52 | 1949.68 | 79.0369 | 290.8734 | -40.2786 | 20187.56 |
| 1 | 200 | -12613734.86 | 8762604.92 | -21468956.81 | -1982.85 | -3351.75 | -155.06 | 79.0515 | -64.1043 | -54.5507 | 20088.94| -15988824.44 | 12956578.08 | -16784515.44 | -3083.30 | -1190.16 | 2021.97 | 79.0515 | -68.3367 | -39.2759 | 20198.11 |
| 2 | 400 | -13004778.82 | 8088552.43 | -21490657.98 | -1927.30 | -3388.29 | -61.92 | 79.0661 | -62.0329 | -54.6558 | 20081.83| -16598590.49 | 12713064.01 | -16373034.29 | -3013.93 | -1244.80 | 2092.55 | 79.0661 | -67.6016 | -38.1371 | 20195.84 |
| 3 | 600 | -13384540.82 | 7407483.02 | -21493715.55 | -1870.04 | -3421.92 | 31.36 | 79.0807 | -59.9500 | -54.6906 | 20074.44| -17194225.99 | 12458727.19 | -15947614.61 | -2942.00 | -1298.38 | 2161.35 | 79.0807 | -66.9149 | -36.9781 | 20193.70 |
| 4 | 800 | -13752681.36 | 6719982.00 | -21478110.75 | -1811.10 | -3452.60 | 124.69 | 79.0952 | -57.8654 | -54.6548 | 20066.78| -17775223.12 | 12193783.59 | -15508617.87 | -2867.56 | -1350.86 | 2228.31 | 79.0952 | -66.2740 | -35.8003 | 20191.68 |
| 5 | 1000 | -14108870.69 | 6026641.36 | -21443841.35 | -1750.53 | -3480.31 | 217.99 | 79.1098 | -55.7894 | -54.5483 | 20058.86| -18341086.49 | 11918458.26 | -15056417.16 | -2790.67 | -1402.19 | 2293.38 | 79.1098 | -65.6762 | -34.6050 | 20189.78 |
| 6 | 1200 | -14452789.10 | 5328059.26 | -21390921.64 | -1688.39 | -3505.01 | 311.18 | 79.1244 | -53.7316 | -54.3719 | 20050.70| -18891333.55 | 11632985.09 | -14591396.85 | -2711.41 | -1452.33 | 2356.50 | 79.1244 | -65.1192 | -33.3933 | 20187.99 |
| 7 | 1400 | -14784127.28 | 4624839.52 | -21319382.57 | -1624.74 | -3526.68 | 404.17 | 79.1390 | -51.7017 | -54.1264 | 20042.33| -19425495.02 | 11337606.67 | -14113952.26 | -2629.83 | -1501.24 | 2417.61 | 79.1390 | -64.6005 | -32.1666 | 20186.31 |
| 8 | 1600 | -15102586.62 | 3917591.05 | -21229271.67 | -1559.62 | -3545.29 | 496.88 | 79.1536 | -49.7083 | -53.8130 | 20033.80| -19943115.25 | 11032574.04 | -13624489.33 | -2546.01 | -1548.87 | 2476.67 | 79.1536 | -64.1178 | -30.9259 | 20184.74 |
| 9 | 1800 | -15407879.51 | 3206927.32 | -21120653.15 | -1493.08 | -3560.83 | 589.23 | 79.1682 | -47.7594 | -53.4336 | 20025.14| -20443752.67 | 10718146.48 | -13123424.30 | -2460.01 | -1595.18 | 2533.62 | 79.1682 | -63.6688 | -29.6721 | 20183.26 |
| 10 | 2000 | -15699729.69 | 2493465.76 | -20993607.83 | -1425.20 | -3573.27 | 681.14 | 79.1828 | -45.8620 | -52.9901 | 20016.40| -20926980.13 | 10394591.33 | -12611183.34 | -2371.92 | -1640.14 | 2588.42 | 79.1828 | -63.2515 | -28.4063 | 20181.87 |
| 11 | 2200 | -15977872.50 | 1777827.24 | -20848233.12 | -1356.02 | -3582.60 | 772.51 | 79.1973 | -44.0223 | -52.4846 | 20007.65| -21392385.29 | 10062183.69 | -12088202.15 | -2281.80 | -1683.70 | 2641.02 | 79.1973 | -62.8637 | -27.1293 | 20180.58 |
| 12 | 2400 | -16242055.18 | 1060635.49 | -20684643.02 | -1285.61 | -3588.80 | 863.28 | 79.2119 | -42.2450 | -51.9197 | 19998.92| -21839570.93 | 9721206.28 | -11554925.67 | -2189.74 | -1725.83 | 2691.37 | 79.2119 | -62.5035 | -25.8419 | 20179.37 |
| 13 | 2600 | -16492037.13 | 342516.47 | -20502968.01 | -1214.02 | -3591.87 | 953.35 | 79.2265 | -40.5342 | -51.2978 | 19990.28| -22268155.36 | 9371949.12 | -11011807.62 | -2095.80 | -1766.49 | 2739.43 | 79.2265 | -62.1691 | -24.5450 | 20178.24 |
| 14 | 2800 | -16727590.20 | -375902.14 | -20303354.99 | -1141.33 | -3591.79 | 1042.64 | 79.2411 | -38.8927 | -50.6218 | 19981.76| -22677772.69 | 9014709.33 | -10459310.18 | -2000.08 | -1805.65 | 2785.16 | 79.2411 | -61.8585 | -23.2392 | 20177.18 |
| 15 | 3000 | -16948498.88 | -1093991.60 | -20085967.19 | -1067.59 | -3588.58 | 1131.08 | 79.2557 | -37.3224 | -49.8942 | 19973.41| -23068073.20 | 8649790.85 | -9897903.54 | -1902.65 | -1843.27 | 2828.51 | 79.2557 | -61.5701 | -21.9253 | 20176.19 |
| 16 | 3200 | -17154560.63 | -1811122.66 | -19850984.05 | -992.87 | -3582.21 | 1218.58 | 79.2703 | -35.8245 | -49.1179 | 19965.28| -23438723.58 | 8277504.20 | -9328065.56 | -1803.59 | -1879.33 | 2869.46 | 79.2703 | -61.3022 | -20.6039 | 20175.27 |
| 17 | 3400 | -17345585.99 | -2526666.17 | -19598601.10 | -917.24 | -3572.70 | 1305.07 | 79.2848 | -34.3991 | -48.2956 | 19957.40| -23789407.26 | 7898166.22 | -8750281.29 | -1703.00 | -1913.78 | 2907.97 | 79.2848 | -61.0532 | -19.2756 | 20174.41 |
| 18 | 3600 | -17521398.89 | -3239993.68 | -19329029.78 | -840.76 | -3560.05 | 1390.46 | 79.2994 | -33.0459 | -47.4300 | 19949.80| -24119824.67 | 7512099.76 | -8165042.65 | -1600.94 | -1946.61 | 2944.00 | 79.2994 | -60.8216 | -17.9410 | 20173.61 |
| 19 | 3800 | -17681836.79 | -3950478.05 | -19042497.30 | -763.50 | -3544.27 | 1474.67 | 79.3140 | -31.7639 | -46.5237 | 19942.50| -24429693.51 | 7119633.47 | -7572847.90 | -1497.52 | -1977.78 | 2977.53 | 79.3140 | -60.6060 | -16.6007 | 20172.87 |
| 20 | 4000 | -17826750.90 | -4657494.04 | -18739246.48 | -685.53 | -3525.37 | 1557.63 | 79.3286 | -30.5517 | -45.5793 | 19935.53| -24718748.97 | 6721101.47 | -6974201.32 | -1392.82 | -2007.26 | 3008.52 | 79.3286 | -60.4049 | -15.2552 | 20172.18 |
| 21 | 4200 | -17956006.32 | -5360418.92 | -18419535.47 | -606.92 | -3503.36 | 1639.26 | 79.3432 | -29.4074 | -44.5992 | 19928.90| -24986743.99 | 6316843.07 | -6369612.69 | -1286.93 | -2035.04 | 3036.94 | 79.3432 | -60.2169 | -13.9050 | 20171.54 |
| 22 | 4400 | -18069482.21 | -6058633.09 | -18083637.62 | -527.75 | -3478.27 | 1719.48 | 79.3578 | -28.3288 | -43.5857 | 19922.63| -25233449.44 | 5907202.51 | -5759596.95 | -1179.94 | -2061.08 | 3062.78 | 79.3578 | -60.0407 | -12.5505 | 20170.94 |
| 23 | 4600 | -18167071.94 | -6751520.65 | -17731841.19 | -448.07 | -3450.10 | 1798.23 | 79.3723 | -27.3137 | -42.5411 | 19916.72| -25458654.36 | 5492528.63 | -5144673.65 | -1071.94 | -2085.37 | 3086.01 | 79.3723 | -59.8751 | -11.1923 | 20170.40 |
| 24 | 4800 | -18248683.21 | -7438470.00 | -17364449.12 | -367.97 | -3418.89 | 1875.43 | 79.3869 | -26.3595 | -41.4674 | 19911.19| -25662166.09 | 5073174.59 | -4525366.61 | -963.03 | -2107.88 | 3106.62 | 79.3869 | -59.7188 | -9.8308 | 20169.89 |
| 25 | 5000 | -18314238.14 | -8118874.50 | -16981778.75 | -287.52 | -3384.66 | 1951.00 | 79.4015 | -25.4636 | -40.3666 | 19906.04| -25843810.49 | 4649497.59 | -3902203.41 | -853.29 | -2128.59 | 3124.57 | 79.4015 | -59.5706 | -8.4664 | 20169.43 |
| 26 | 5200 | -18363673.43 | -8792132.97 | -16584161.54 | -206.79 | -3347.43 | 2024.88 | 79.4161 | -24.6233 | -39.2407 | 19901.27| -26003432.06 | 4221858.52 | -3275714.94 | -742.82 | -2147.49 | 3139.87 | 79.4161 | -59.4293 | -7.0995 | 20169.00 |
| 27 | 5400 | -18396940.35 | -9457650.35 | -16171942.77 | -125.85 | -3307.25 | 2097.01 | 79.4307 | -23.8359 | -38.0913 | 19896.88| -26140894.10 | 3790621.68 | -2646434.98 | -631.71 | -2164.57 | 3152.49 | 79.4307 | -59.2938 | -5.7306 | 20168.62 |
| 28 | 5600 | -18414004.90 | -10114838.25 | -15745481.22 | -44.78 | -3264.14 | 2167.30 | 79.4453 | -23.0986 | -36.9201 | 19892.88| -26256078.78 | 3356154.45 | -2014899.72 | -520.06 | -2179.80 | 3162.42 | 79.4453 | -59.1629 | -4.3600 | 20168.27 |
| 29 | 5800 | -18414847.80 | -10763115.54 | -15305148.84 | 36.35 | -3218.15 | 2235.70 | 79.4599 | -22.4088 | -35.7286 | 19889.26| -26348887.31 | 2918827.03 | -1381647.32 | -407.96 | -2193.17 | 3169.66 | 79.4599 | -59.0355 | -2.9882 | 20167.96 |
| 30 | 6000 | -18399464.52 | -11401908.94 | -14851330.41 | 117.47 | -3169.31 | 2302.15 | 79.4744 | -21.7639 | -34.5183 | 19886.02| -26419239.97 | 2479012.03 | -747217.40 | -295.51 | -2204.67 | 3174.19 | 79.4744 | -58.9104 | -1.6156 | 20167.69 |
| 31 | 6200 | -18367865.34 | -12030653.54 | -14384423.14 | 198.50 | -3117.67 | 2366.58 | 79.4890 | -21.1613 | -33.2906 | 19883.15| -26467076.21 | 2037084.23 | -112150.66 | -182.81 | -2214.29 | 3176.02 | 79.4890 | -58.7867 | -0.2425 | 20167.45 |
| 32 | 6400 | -18320075.32 | -12648793.39 | -13904836.36 | 279.37 | -3063.27 | 2428.94 | 79.5036 | -20.5986 | -32.0467 | 19880.65| -26492354.72 | 1593420.25 | 523011.67 | -69.95 | -2222.03 | 3175.15 | 79.5036 | -58.6631 | 1.1308 | 20167.25 |
| 33 | 6600 | -18256134.31 | -13255782.03 | -13412991.06 | 360.00 | -3006.17 | 2489.16 | 79.5182 | -20.0733 | -30.7879 | 19878.52| -26495053.40 | 1148398.16 | 1157728.23 | 42.97 | -2227.87 | 3171.57 | 79.5182 | -58.5386 | 2.5037 | 20167.09 |
| 34 | 6800 | -18176096.87 | -13851083.06 | -12909319.55 | 440.32 | -2946.41 | 2547.19 | 79.5328 | -19.5831 | -29.5151 | 19876.76| -26475169.48 | 702397.27 | 1791458.01 | 155.86 | -2231.82 | 3165.28 | 79.5328 | -58.4120 | 3.8759 | 20166.97 |
| 35 | 7000 | -18080032.29 | -14434170.60 | -12394264.98 | 520.26 | -2884.04 | 2602.98 | 79.5474 | -19.1259 | -28.2295 | 19875.35| -26432719.41 | 255797.68 | 2423660.84 | 268.61 | -2233.86 | 3156.30 | 79.5474 | -58.2824 | 5.2471 | 20166.88 |
| 36 | 7200 | -17968024.48 | -15004529.88 | -11868280.98 | 599.74 | -2819.13 | 2656.47 | 79.5619 | -18.6994 | -26.9321 | 19874.29| -26367738.96 | -191019.94 | 3053797.82 | 381.14 | -2234.00 | 3144.62 | 79.5619 | -58.1485 | 6.6168 | 20166.83 |
| 37 | 7400 | -17840171.90 | -15561657.70 | -11331831.16 | 678.69 | -2751.74 | 2707.63 | 79.5765 | -18.3016 | -25.6238 | 19873.59| -26280283.11 | -637674.73 | 3681331.80 | 493.35 | -2232.23 | 3130.27 | 79.5765 | -58.0092 | 7.9847 | 20166.82 |
| 38 | 7600 | -17696587.50 | -16105062.93 | -10785388.69 | 757.05 | -2681.92 | 2756.39 | 79.5911 | -17.9305 | -24.3054 | 19873.22| -26170426.03 | -1083785.97 | 4305727.81 | 605.14 | -2228.56 | 3113.25 | 79.5911 | -57.8634 | 9.3503 | 20166.85 |
| 39 | 7800 | -17537398.54 | -16634266.99 | -10229435.85 | 834.73 | -2609.74 | 2802.73 | 79.6057 | -17.5843 | -22.9778 | 19873.20| -26038261.04 | -1528973.38 | 4926453.59 | 716.42 | -2222.99 | 3093.57 | 79.6057 | -57.7098 | 10.7132 | 20166.93 |
| 40 | 8000 | -17362746.53 | -17148804.32 | -9664463.51 | 911.66 | -2535.26 | 2846.58 | 79.6203 | -17.2612 | -21.6417 | 19873.50| -25883900.49 | -1972857.47 | 5542979.95 | 827.08 | -2215.53 | 3071.26 | 79.6203 | -57.5474 | 12.0730 | 20167.05 |
| 41 | 8200 | -17172787.05 | -17648222.81 | -9090970.72 | 987.79 | -2458.56 | 2887.92 | 79.6349 | -16.9594 | -20.2978 | 19874.13| -25707475.70 | -2415059.82 | 6154781.30 | 937.04 | -2206.18 | 3046.32 | 79.6349 | -57.3748 | 13.4292 | 20167.22 |
| 42 | 8400 | -16967689.62 | -18132084.26 | -8509464.15 | 1063.03 | -2379.70 | 2926.71 | 79.6494 | -16.6772 | -18.9470 | 19875.09| -25509136.83 | -2855203.49 | 6761336.07 | 1046.21 | -2194.95 | 3018.79 | 79.6494 | -57.1908 | 14.7814 | 20167.44 |
| 43 | 8600 | -16747637.49 | -18599964.78 | -7920457.63 | 1137.32 | -2298.76 | 2962.92 | 79.6640 | -16.4131 | -17.5897 | 19876.36| -25289052.76 | -3292913.23 | 7362127.15 | 1154.48 | -2181.84 | 2988.69 | 79.6640 | -56.9941 | 16.1291 | 20167.72 |
| 44 | 8800 | -16512827.50 | -19051455.20 | -7324471.67 | 1210.60 | -2215.81 | 2996.50 | 79.6786 | -16.1655 | -16.2267 | 19877.94| -25047410.92 | -3727815.93 | 7956642.36 | 1261.77 | -2166.88 | 2956.04 | 79.6786 | -56.7833 | 17.4719 | 20168.05 |
| 45 | 9000 | -16263469.83 | -19486161.46 | -6722032.89 | 1282.79 | -2130.93 | 3027.44 | 79.6932 | -15.9330 | -14.8585 | 19879.83| -24784417.19 | -4159540.81 | 8544374.85 | 1367.98 | -2150.07 | 2920.87 | 79.6932 | -56.5570 | 18.8091 | 20168.43 |
| 46 | 9200 | -15999787.82 | -19903704.96 | -6113673.53 | 1353.83 | -2044.20 | 3055.70 | 79.7078 | -15.7141 | -13.4857 | 19882.01| -24500295.65 | -4587719.83 | 9124823.58 | 1473.03 | -2131.42 | 2883.21 | 79.7078 | -56.3138 | 20.1402 | 20168.88 |
| 47 | 9400 | -15722017.73 | -20303722.95 | -5499930.96 | 1423.66 | -1955.69 | 3081.27 | 79.7224 | -15.5074 | -12.1089 | 19884.50| -24195288.46 | -5011987.98 | 9697493.71 | 1576.82 | -2110.96 | 2843.09 | 79.7224 | -56.0522 | 21.4647 | 20169.40 |
| 48 | 9600 | -15430408.48 | -20685868.83 | -4881347.11 | 1492.21 | -1865.49 | 3104.11 | 79.7370 | -15.3116 | -10.7285 | 19887.27| -23869655.58 | -5431983.58 | 10261897.04 | 1679.27 | -2088.70 | 2800.54 | 79.7370 | -55.7705 | 22.7820 | 20169.99 |
| 49 | 9800 | -15125221.44 | -21049812.46 | -4258467.94 | 1559.43 | -1773.68 | 3124.22 | 79.7515 | -15.1255 | -9.3451 | 19890.33| -23523674.62 | -5847348.59 | 10817552.45 | 1780.29 | -2064.66 | 2755.61 | 79.7515 | -55.4673 | 24.0914 | 20170.65 |
| 50 | 10000 | -14806730.09 | -21395240.51 | -3631842.96 | 1625.25 | -1680.35 | 3141.57 | 79.7661 | -14.9476 | -7.9592 | 19893.68| -23157640.53 | -6257728.94 | 11363986.28 | 1879.79 | -2038.86 | 2708.34 | 79.7661 | -55.1407 | 25.3922 | 20171.39 |
| 51 | 10200 | -14475219.80 | -21721856.65 | -3002024.61 | 1689.61 | -1585.58 | 3156.15 | 79.7807 | -14.7769 | -6.5713 | 19897.29| -22771865.43 | -6662774.81 | 11900732.72 | 1977.69 | -2011.32 | 2658.75 | 79.7807 | -54.7889 | 26.6837 | 20172.22 |
| 52 | 10400 | -14130987.50 | -22029381.91 | -2369567.80 | 1752.46 | -1489.46 | 3167.95 | 79.7953 | -14.6120 | -5.1818 | 19901.18| -22366678.24 | -7062140.94 | 12427334.29 | 2073.90 | -1982.06 | 2606.89 | 79.7953 | -54.4102 | 27.9652 | 20173.14 |
| 53 | 10600 | -13774341.40 | -22317554.84 | -1735029.33 | 1813.74 | -1392.07 | 3176.97 | 79.8099 | -14.4518 | -3.7912 | 19905.34| -21942424.51 | -7455486.92 | 12943342.13 | 2168.34 | -1951.12 | 2552.82 | 79.8099 | -54.0025 | 29.2358 | 20174.15 |
| 54 | 10800 | -13405600.66 | -22586131.76 | -1098967.36 | 1873.40 | -1293.51 | 3183.19 | 79.8245 | -14.2951 | -2.3999 | 19909.75| -21499466.02 | -7842477.50 | 13448316.46 | 2260.93 | -1918.51 | 2496.57 | 79.8245 | -53.5639 | 30.4946 | 20175.25 |
| 55 | 11000 | -13025095.05 | -22834886.97 | -461940.89 | 1931.38 | -1193.87 | 3186.61 | 79.8390 | -14.1408 | -1.0083 | 19914.43| -21038180.54 | -8222782.84 | 13941826.92 | 2351.60 | -1884.27 | 2438.19 | 79.8390 | -53.0922 | 31.7408 | 20176.47 |
| 56 | 11200 | -12633164.63 | -23063612.92 | 175490.82 | 1987.63 | -1093.23 | 3187.24 | 79.8536 | -13.9877 | 0.3830 | 19919.35| -20558961.49 | -8596078.84 | 14423452.95 | 2440.25 | -1848.42 | 2377.73 | 79.8536 | -52.5851 | 32.9734 | 20177.79 |
| 57 | 11400 | -12230159.42 | -23272120.35 | 812768.75 | 2042.12 | -991.70 | 3185.07 | 79.8682 | -13.8347 | 1.7736 | 19924.53| -20062217.58 | -8962047.37 | 14892784.14 | 2526.83 | -1811.00 | 2315.25 | 79.8682 | -52.0405 | 34.1912 | 20179.23 |
| 58 | 11600 | -11816439.00 | -23460238.46 | 1449334.64 | 2094.78 | -889.36 | 3180.12 | 79.8828 | -13.6805 | 3.1632 | 19929.94| -19548372.47 | -9320376.59 | 15349420.61 | 2611.25 | -1772.04 | 2250.79 | 79.8828 | -51.4560 | 35.3932 | 20180.79 |
| 59 | 11800 | -11392372.18 | -23627815.03 | 2084631.56 | 2145.57 | -786.30 | 3172.39 | 79.8974 | -13.5242 | 4.5513 | 19935.60| -19017864.43 | -9670761.16 | 15792973.32 | 2693.45 | -1731.56 | 2184.42 | 79.8974 | -50.8289 | 36.5781 | 20182.47 |
| 60 | 12000 | -10958336.62 | -23774716.49 | 2718104.42 | 2194.46 | -682.62 | 3161.88 | 79.9120 | -13.3644 | 5.9374 | 19941.49| -18471145.93 | -10012902.56 | 16223064.39 | 2773.35 | -1689.61 | 2116.19 | 79.9120 | -50.1569 | 37.7446 | 20184.29 |
| 61 | 12200 | -10514718.43 | -23900828.03 | 3349200.48 | 2241.39 | -578.42 | 3148.62 | 79.9265 | -13.2001 | 7.3211 | 19947.61| -17908683.25 | -10346509.31 | 16639327.49 | 2850.88 | -1646.22 | 2046.15 | 79.9265 | -49.4373 | 38.8913 | 20186.24 |
| 62 | 12400 | -10061911.83 | -24006053.64 | 3977369.88 | 2286.34 | -473.78 | 3132.62 | 79.9411 | -13.0301 | 8.7020 | 19953.95| -17330956.13 | -10671297.22 | 17041408.08 | 2925.98 | -1601.43 | 1974.37 | 79.9411 | -48.6674 | 40.0168 | 20188.33 |
| 63 | 12600 | -9600318.69 | -24090316.18 | 4602066.14 | 2329.25 | -368.80 | 3113.89 | 79.9557 | -12.8531 | 10.0795 | 19960.52| -16738457.29 | -10986989.66 | 17428963.74 | 2998.59 | -1555.27 | 1900.91 | 79.9557 | -47.8446 | 41.1195 | 20190.56 |
| 64 | 12800 | -9130348.20 | -24153557.33 | 5222746.67 | 2370.11 | -263.58 | 3092.46 | 79.9703 | -12.6680 | 11.4533 | 19967.31| -16131692.07 | -11293317.75 | 17801664.49 | 3068.63 | -1507.79 | 1825.83 | 79.9703 | -46.9660 | 42.1975 | 20192.93 |
| 65 | 13000 | -8652416.41 | -24195737.70 | 5838873.27 | 2408.86 | -158.21 | 3068.36 | 79.9849 | -12.4735 | 12.8229 | 19974.31| -15511177.92 | -11590020.65 | 18159193.03 | 3136.07 | -1459.03 | 1749.20 | 79.9849 | -46.0291 | 43.2492 | 20195.44 |
| 66 | 13200 | -8166945.85 | -24216836.70 | 6449912.62 | 2445.49 | -52.78 | 3041.60 | 79.9995 | -12.2683 | 14.1877 | 19981.52| -14877444.06 | -11876845.70 | 18501245.02 | 3200.82 | -1409.02 | 1671.08 | 79.9995 | -45.0312 | 44.2726 | 20198.09 |
| 67 | 13400 | -7674365.10 | -24216852.58 | 7055336.77 | 2479.96 | 52.61 | 3012.21 | 80.0141 | -12.0511 | 15.5473 | 19988.94| -14231030.91 | -12153548.74 | 18827529.35 | 3262.85 | -1357.81 | 1591.53 | 80.0141 | -43.9697 | 45.2657 | 20200.88 |
| 68 | 13600 | -7175108.41 | -24195802.33 | 7654623.58 | 2512.24 | 157.86 | 2980.23 | 80.0286 | -11.8206 | 16.9011 | 19996.56| -13572489.73 | -12419894.22 | 19137768.40 | 3322.09 | -1305.45 | 1510.64 | 80.0286 | -42.8423 | 46.2262 | 20203.79 |
| 69 | 13800 | -6669615.21 | -24153721.63 | 8247257.23 | 2542.32 | 262.90 | 2945.68 | 80.0432 | -11.5754 | 18.2486 | 20004.39| -12902382.06 | -12675655.47 | 19431698.25 | 3378.51 | -1251.98 | 1428.45 | 80.0432 | -41.6467 | 47.1519 | 20206.82 |
| 70 | 14000 | -6158329.75 | -24090664.75 | 8832728.67 | 2570.16 | 367.61 | 2908.61 | 80.0578 | -11.3140 | 19.5892 | 20012.41| -12221279.31 | -12920614.85 | 19709068.90 | 3432.04 | -1197.44 | 1345.06 | 80.0578 | -40.3812 | 48.0404 | 20209.96 |
| 71 | 14200 | -5641700.66 | -24006704.40 | 9410536.03 | 2595.75 | 471.92 | 2869.05 | 80.0724 | -11.0349 | 20.9223 | 20020.62| -11529762.23 | -13154563.96 | 19969644.51 | 3482.64 | -1141.88 | 1260.52 | 80.0724 | -39.0441 | 48.8892 | 20213.18 |
| 72 | 14400 | -5120180.48 | -23901931.68 | 9980185.09 | 2619.07 | 575.72 | 2827.04 | 80.0870 | -10.7366 | 22.2473 | 20029.02| -10828420.40 | -13377303.81 | 20213203.59 | 3530.28 | -1085.36 | 1174.90 | 80.0870 | -37.6345 | 49.6958 | 20216.47 |
| 73 | 14600 | -4594225.30 | -23776455.85 | 10541189.72 | 2640.10 | 678.93 | 2782.61 | 80.1016 | -10.4175 | 23.5636 | 20037.61| -10117851.77 | -13588644.97 | 20439539.19 | 3574.90 | -1027.91 | 1088.29 | 80.1016 | -36.1520 | 50.4574 | 20219.79 |
| 74 | 14800 | -4064294.27 | -23630404.23 | 11093072.23 | 2658.83 | 781.46 | 2735.82 | 80.1161 | -10.0760 | 24.8704 | 20046.39| -9398662.10 | -13788407.78 | 20648459.04 | 3616.48 | -969.58 | 1000.76 | 80.1161 | -34.5966 | 51.1713 | 20223.11 |
| 75 | 15000 | -3530849.23 | -23463921.98 | 11635363.86 | 2675.24 | 883.22 | 2686.71 | 80.1307 | -9.7102 | 26.1671 | 20055.35| -8671464.49 | -13976422.45 | 20839785.78 | 3654.98 | -910.43 | 912.37 | 80.1307 | -32.9695 | 51.8348 | 20226.41 |
| 76 | 15200 | -2994354.22 | -23277171.92 | 12167605.10 | 2689.32 | 984.13 | 2635.33 | 80.1453 | -9.3184 | 27.4528 | 20064.49| -7936878.78 | -14152529.21 | 21013357.04 | 3690.36 | -850.51 | 823.22 | 80.1453 | -31.2723 | 52.4452 | 20229.63 |
| 77 | 15400 | -2455275.09 | -23070334.30 | 12689346.12 | 2701.08 | 1084.08 | 2581.72 | 80.1599 | -8.8986 | 28.7267 | 20073.81| -7195531.13 | -14316578.50 | 21169025.63 | 3722.59 | -789.87 | 733.36 | 80.1599 | -29.5079 | 52.9998 | 20232.73 |
| 78 | 15600 | -1914079.08 | -22843606.60 | 13200147.12 | 2710.49 | 1183.01 | 2525.93 | 80.1745 | -8.4490 | 29.9881 | 20083.30| -6448053.35 | -14468431.02 | 21306659.62 | 3751.65 | -728.55 | 642.88 | 80.1745 | -27.6802 | 53.4960 | 20235.67 |
| 79 | 15800 | -1371234.37 | -22597203.28 | 13699578.67 | 2717.56 | 1280.83 | 2468.03 | 80.1891 | -7.9675 | 31.2358 | 20092.96| -5695082.49 | -14607957.91 | 21426142.50 | 3777.52 | -666.62 | 551.86 | 80.1891 | -25.7939 | 53.9313 | 20238.40 |
| 80 | 16000 | -827209.66 | -22331355.50 | 14187222.09 | 2722.29 | 1377.44 | 2408.06 | 80.2036 | -7.4520 | 32.4691 | 20102.80| -4937260.18 | -14735040.80 | 21527373.23 | 3800.17 | -604.12 | 460.37 | 80.2036 | -23.8550 | 54.3035 | 20240.87 |
| 81 | 16200 | -282473.77 | -22046310.88 | 14662669.76 | 2724.68 | 1472.78 | 2346.08 | 80.2182 | -6.9003 | 33.6867 | 20112.80| -4175232.18 | -14849571.98 | 21610266.34 | 3819.57 | -541.11 | 368.50 | 80.2182 | -21.8705 | 54.6106 | 20243.04 |
| 82 | 16400 | 262504.77 | -21742333.20 | 15125525.42 | 2724.72 | 1566.76 | 2282.15 | 80.2328 | -6.3101 | 34.8877 | 20122.97| -3409647.74 | -14951454.43 | 21674752.02 | 3835.73 | -477.64 | 276.31 | 80.2328 | -19.8483 | 54.8508 | 20244.86 |
| 83 | 16600 | 807258.19 | -21419702.10 | 15575404.53 | 2722.43 | 1659.30 | 2216.33 | 80.2474 | -5.6791 | 36.0709 | 20133.30| -2641159.12 | -15040601.92 | 21720776.15 | 3848.61 | -413.77 | 183.90 | 80.2474 | -17.7972 | 55.0227 | 20246.30 |
| 84 | 16800 | 1351319.80 | -21078712.78 | 16011934.50 | 2717.80 | 1750.33 | 2148.67 | 80.2620 | -5.0050 | 37.2349 | 20143.79| -1870420.98 | -15116939.08 | 21748300.34 | 3858.22 | -349.55 | 91.33 | 80.2620 | -15.7264 | 55.1254 | 20247.33 |
| 85 | 17000 | 1894224.49 | -20719675.69 | 16434755.01 | 2710.86 | 1839.77 | 2079.24 | 80.2766 | -4.2851 | 38.3784 | 20154.44| -1098089.83 | -15180401.50 | 21757302.00 | 3864.54 | -285.03 | -1.32 | 80.2766 | -13.6460 | 55.1582 | 20247.94 |
| 86 | 17200 | 2435509.04 | -20342916.16 | 16843518.27 | 2701.60 | 1927.54 | 2008.11 | 80.2912 | -3.5172 | 39.5000 | 20165.23| -324823.51 | -15230935.71 | 21747774.32 | 3867.57 | -220.28 | -93.95 | 80.2912 | -11.5660 | 55.1209 | 20248.13 |
| 87 | 17400 | 2974712.60 | -19948774.11 | 17237889.26 | 2690.05 | 2013.58 | 1935.33 | 80.3057 | -2.6986 | 40.5982 | 20176.17| 448719.44 | -15268499.29 | 21719726.28 | 3867.31 | -155.33 | -186.51 | 80.3057 | -9.4966 | 55.0137 | 20247.89 |
| 88 | 17600 | 3511377.01 | -19537603.64 | 17617545.97 | 2676.21 | 2097.82 | 1860.98 | 80.3203 | -1.8269 | 41.6713 | 20187.25| 1221880.28 | -15293060.86 | 21673182.64 | 3863.75 | -90.26 | -278.90 | 80.3203 | -7.4474 | 54.8373 | 20247.25 |
| 89 | 17800 | 4045047.22 | -19109772.70 | 17982179.62 | 2660.11 | 2180.18 | 1785.11 | 80.3349 | -0.8995 | 42.7177 | 20198.46| 1994000.68 | -15304600.13 | 21608183.92 | 3856.90 | -25.12 | -371.04 | 80.3349 | -5.4280 | 54.5928 | 20246.22 |
| 90 | 18000 | 4575271.67 | -18665662.73 | 18331494.92 | 2641.76 | 2260.60 | 1707.81 | 80.3495 | 0.0859 | 43.7354 | 20209.79| 2764423.23 | -15303107.92 | 21524786.37 | 3846.77 | 40.04 | -462.87 | 80.3495 | -3.4470 | 54.2816 | 20244.85 |
| 91 | 18200 | 5101602.65 | -18205668.24 | 18665210.19 | 2621.18 | 2339.01 | 1629.12 | 80.3641 | 1.1316 | 44.7226 | 20221.23| 3532492.08 | -15288586.13 | 21423061.91 | 3833.37 | 105.17 | -554.30 | 80.3641 | -1.5123 | 53.9054 | 20243.18 |
| 92 | 18400 | 5623596.66 | -17730196.45 | 18983057.60 | 2598.39 | 2415.36 | 1549.14 | 80.3787 | 2.2398 | 45.6773 | 20232.76| 4297553.39 | -15261047.78 | 21303098.04 | 3816.70 | 170.20 | -645.25 | 80.3787 | 0.3694 | 53.4664 | 20241.24 |
| 93 | 18600 | 6140814.80 | -17239666.89 | 19284783.33 | 2573.43 | 2489.58 | 1467.92 | 80.3932 | 3.4124 | 46.5972 | 20244.36| 5058956.00 | -15220516.97 | 21164997.81 | 3796.78 | 235.08 | -735.65 | 80.3932 | 2.1920 | 52.9666 | 20239.09 |
| 94 | 18800 | 6652823.09 | -16734510.96 | 19570147.72 | 2546.30 | 2561.61 | 1385.54 | 80.4078 | 4.6511 | 47.4802 | 20256.03| 5816051.90 | -15167028.84 | 21008879.72 | 3773.64 | 299.76 | -825.42 | 80.4078 | 3.9509 | 52.4088 | 20236.78 |
| 95 | 19000 | 7159192.83 | -16215171.57 | 19838925.41 | 2517.04 | 2631.40 | 1302.06 | 80.4224 | 5.9572 | 48.3239 | 20267.72| 6568196.82 | -15100629.61 | 20834877.56 | 3747.28 | 364.18 | -914.48 | 80.4224 | 5.6424 | 51.7954 | 20234.34 |
| 96 | 19200 | 7659500.95 | -15682102.71 | 20090905.50 | 2485.69 | 2698.90 | 1217.57 | 80.4370 | 7.3315 | 49.1259 | 20279.43| 7314750.78 | -15021376.44 | 20643140.38 | 3717.73 | 428.29 | -1002.76 | 80.4370 | 7.2636 | 51.1291 | 20231.83 |
| 97 | 19400 | 8153330.31 | -15135768.98 | 20325891.62 | 2452.26 | 2764.04 | 1132.14 | 80.4516 | 8.7744 | 49.8839 | 20291.10| 8055078.61 | -14929337.47 | 20433832.28 | 3685.02 | 492.03 | -1090.18 | 80.4516 | 8.8129 | 50.4127 | 20229.28 |
| 98 | 19600 | 8640270.05 | -14576645.22 | 20543702.09 | 2416.80 | 2826.79 | 1045.83 | 80.4662 | 10.2856 | 50.5952 | 20302.71| 8788550.53 | -14824591.68 | 20207132.33 | 3649.18 | 555.35 | -1176.66 | 80.4662 | 10.2893 | 49.6489 | 20226.73 |
| 99 | 19800 | 9119915.90 | -14005216.05 | 20744169.97 | 2379.33 | 2887.09 | 958.72 | 80.4807 | 11.8641 | 51.2574 | 20314.22| 9514542.64 | -14707228.89 | 19963234.37 | 3610.23 | 618.19 | -1262.14 | 80.4807 | 11.6927 | 48.8405 | 20224.21 |
| 100 | 20000 | 9591870.47 | -13421975.41 | 20927143.16 | 2339.89 | 2944.90 | 870.89 | 80.4953 | 13.5082 | 51.8680 | 20325.58| 10232437.49 | -14577349.63 | 19702346.86 | 3568.21 | 680.51 | -1346.55 | 80.4953 | 13.0237 | 47.9900 | 20221.75 |
| 101 | 20200 | 10055743.60 | -12827426.17 | 21092484.45 | 2298.52 | 3000.17 | 782.42 | 80.5099 | 15.2153 | 52.4245 | 20336.74| 10941624.56 | -14435065.08 | 19424692.70 | 3523.16 | 742.24 | -1429.80 | 80.5099 | 14.2832 | 47.1001 | 20219.36 |
| 102 | 20400 | 10511152.57 | -12222079.63 | 21240071.61 | 2255.26 | 3052.86 | 693.37 | 80.5245 | 16.9818 | 52.9246 | 20347.67| 11641500.83 | -14280496.97 | 19130509.04 | 3475.11 | 803.33 | -1511.83 | 80.5245 | 15.4729 | 46.1733 | 20217.07 |
| 103 | 20600 | 10957722.47 | -11606455.10 | 21369797.39 | 2210.14 | 3102.94 | 603.82 | 80.5391 | 18.8034 | 53.3660 | 20358.30| 12331471.25 | -14113777.46 | 18820047.07 | 3424.11 | 863.74 | -1592.57 | 80.5391 | 16.5946 | 45.2119 | 20214.88 |
| 104 | 20800 | 11395086.41 | -10981079.45 | 21481569.57 | 2163.20 | 3150.37 | 513.84 | 80.5537 | 20.6746 | 53.7468 | 20368.59| 13010949.27 | -13935049.05 | 18493571.79 | 3370.19 | 923.41 | -1671.95 | 80.5537 | 17.6505 | 44.2182 | 20212.81 |
| 105 | 21000 | 11822885.82 | -10346486.63 | 21575310.98 | 2114.50 | 3195.11 | 423.52 | 80.5683 | 22.5891 | 54.0650 | 20378.49| 13679357.33 | -13744464.42 | 18151361.82 | 3313.41 | 982.30 | -1749.91 | 80.5683 | 18.6430 | 43.1944 | 20210.86 |
| 106 | 21200 | 12240770.69 | -9703217.25 | 21650959.50 | 2064.07 | 3237.13 | 332.93 | 80.5828 | 24.5397 | 54.3190 | 20387.96| 14336127.32 | -13542186.34 | 17793709.11 | 3253.82 | 1040.34 | -1826.37 | 80.5828 | 19.5747 | 42.1426 | 20209.04 |
| 107 | 21400 | 12648399.85 | -9051818.06 | 21708468.05 | 2011.95 | 3276.40 | 242.13 | 80.5974 | 26.5184 | 54.5077 | 20396.96| 14980701.14 | -13328387.49 | 17420918.73 | 3191.46 | 1097.50 | -1901.27 | 80.5974 | 20.4482 | 41.0645 | 20207.34 |
| 108 | 21600 | 13045441.17 | -8392841.58 | 21747804.60 | 1958.20 | 3312.90 | 151.22 | 80.6120 | 28.5168 | 54.6300 | 20405.45| 15612531.06 | -13103250.35 | 17033308.59 | 3126.39 | 1153.72 | -1974.55 | 80.6120 | 21.2662 | 39.9622 | 20205.77 |
| 109 | 21800 | 13431571.81 | -7726845.57 | 21768952.10 | 1902.85 | 3346.59 | 60.26 | 80.6266 | 30.5258 | 54.6854 | 20413.42| 16231080.28 | -12866967.02 | 16631209.18 | 3058.66 | 1208.95 | -2046.15 | 80.6266 | 22.0314 | 38.8371 | 20204.33 |
| 110 | 22000 | 13806478.46 | -7054392.56 | 21771908.47 | 1845.96 | 3377.46 | -30.68 | 80.6412 | 32.5363 | 54.6736 | 20420.84| 16835823.35 | -12619739.03 | 16214963.25 | 2988.34 | 1263.15 | -2116.01 | 80.6412 | 22.7466 | 37.6910 | 20203.01 |
| 111 | 22200 | 14169857.54 | -6376049.47 | 21756686.55 | 1787.58 | 3405.49 | -121.52 | 80.6558 | 34.5389 | 54.5950 | 20427.73| 17426246.60 | -12361777.24 | 15784925.56 | 2915.48 | 1316.28 | -2184.06 | 80.6558 | 23.4143 | 36.5253 | 20201.81 |
| 112 | 22400 | 14521415.40 | -5692387.05 | 21723314.01 | 1727.76 | 3430.65 | -212.17 | 80.6703 | 36.5246 | 54.4499 | 20434.08| 18001848.57 | -12093301.57 | 15341462.57 | 2840.14 | 1368.29 | -2250.25 | 80.6703 | 24.0373 | 35.3414 | 20200.72 |
| 113 | 22600 | 14860868.51 | -5003979.49 | 21671833.33 | 1666.54 | 3452.94 | -302.58 | 80.6849 | 38.4846 | 54.2393 | 20439.90| 18562140.49 | -11814540.89 | 14884952.09 | 2762.39 | 1419.12 | -2314.53 | 80.6849 | 24.6179 | 34.1406 | 20199.74 |
| 114 | 22800 | 15187943.70 | -4311403.93 | 21602301.66 | 1603.99 | 3472.33 | -392.67 | 80.6995 | 40.4108 | 53.9646 | 20445.23| 19106646.61 | -11525732.77 | 14415783.00 | 2682.29 | 1468.75 | -2376.83 | 80.6995 | 25.1588 | 32.9240 | 20198.86 |
| 115 | 23000 | 15502378.27 | -3615239.98 | 21514790.78 | 1540.15 | 3488.82 | -482.36 | 80.7141 | 42.2958 | 53.6272 | 20450.08| 19634904.67 | -11227123.30 | 13934354.90 | 2599.92 | 1517.13 | -2437.11 | 80.7141 | 25.6622 | 31.6929 | 20198.08 |
| 116 | 23200 | 15803920.21 | -2916069.33 | 21409386.97 | 1475.07 | 3502.40 | -571.59 | 80.7287 | 44.1329 | 53.2291 | 20454.50| 20146466.25 | -10918966.88 | 13441077.76 | 2515.34 | 1564.22 | -2495.31 | 80.7287 | 26.1305 | 30.4483 | 20197.39 |
| 117 | 23400 | 16092328.35 | -2214475.21 | 21286190.89 | 1408.82 | 3513.06 | -660.28 | 80.7433 | 45.9163 | 52.7723 | 20458.52| 20640897.17 | -10601525.99 | 12936371.58 | 2428.62 | 1609.97 | -2551.39 | 80.7433 | 26.5658 | 29.1911 | 20196.79 |
| 118 | 23600 | 16367372.50 | -1511041.99 | 21145317.49 | 1341.44 | 3520.79 | -748.35 | 80.7579 | 47.6414 | 52.2590 | 20462.17| 21117777.83 | -10275070.98 | 12420666.03 | 2339.85 | 1654.35 | -2605.30 | 80.7579 | 26.9703 | 27.9223 | 20196.26 |
| 119 | 23800 | 16628833.61 | -806354.71 | 20986895.87 | 1273.00 | 3525.60 | -835.75 | 80.7724 | 49.3042 | 51.6916 | 20465.49| 21576703.59 | -9939879.82 | 11894400.10 | 2249.09 | 1697.32 | -2656.99 | 80.7724 | 27.3460 | 26.6427 | 20195.81 |
| 120 | 24000 | 16876503.93 | -100998.63 | 20811069.13 | 1203.54 | 3527.48 | -922.39 | 80.7870 | 50.9019 | 51.0727 | 20468.52| 22017285.10 | -9596237.90 | 11358021.69 | 2156.42 | 1738.85 | -2706.42 | 80.7870 | 27.6948 | 25.3532 | 20195.43 |
| 121 | 24200 | 17110187.07 | 604441.22 | 20617994.23 | 1133.13 | 3526.43 | -1008.22 | 80.8016 | 52.4324 | 50.4047 | 20471.29| 22439148.63 | -9244437.74 | 10811987.26 | 2061.92 | 1778.90 | -2753.54 | 80.8016 | 28.0186 | 24.0545 | 20195.11 |
| 122 | 24400 | 17329698.20 | 1309380.48 | 20407841.83 | 1061.83 | 3522.47 | -1093.15 | 80.8162 | 53.8945 | 49.6903 | 20473.83| 22841936.37 | -8884778.76 | 10256761.43 | 1965.67 | 1817.43 | -2798.32 | 80.8162 | 28.3191 | 22.7474 | 20194.85 |
| 123 | 24600 | 17534864.11 | 2013235.91 | 20180796.14 | 989.69 | 3515.60 | -1177.14 | 80.8308 | 55.2876 | 48.9320 | 20476.17| 23225306.75 | -8517567.06 | 9692816.57 | 1867.76 | 1854.42 | -2840.73 | 80.8308 | 28.5981 | 21.4324 | 20194.64 |
| 124 | 24800 | 17725523.33 | 2715425.84 | 19937054.72 | 916.78 | 3505.82 | -1260.10 | 80.8454 | 56.6119 | 48.1322 | 20478.33| 23588934.71 | -8143115.09 | 9120632.44 | 1768.26 | 1889.83 | -2880.71 | 80.8454 | 28.8572 | 20.1103 | 20194.48 |
| 125 | 25000 | 17901526.21 | 3415370.56 | 19676828.33 | 843.14 | 3493.15 | -1341.98 | 80.8599 | 57.8682 | 47.2935 | 20480.33| 23932512.00 | -7761741.43 | 8540695.74 | 1667.27 | 1923.63 | -2918.25 | 80.8599 | 29.0979 | 18.7816 | 20194.36 |
| 126 | 25200 | 18062735.05 | 4112492.83 | 19400340.74 | 768.84 | 3477.60 | -1422.70 | 80.8745 | 59.0575 | 46.4181 | 20482.17| 24255747.39 | -7373770.54 | 7953499.70 | 1564.86 | 1955.80 | -2953.30 | 80.8745 | 29.3218 | 17.4469 | 20194.29 |
| 127 | 25400 | 18209024.11 | 4806218.24 | 19107828.48 | 693.95 | 3459.18 | -1502.21 | 80.8891 | 60.1813 | 45.5085 | 20483.88| 24558366.97 | -6979532.41 | 7359543.70 | 1461.12 | 1986.30 | -2985.84 | 80.8891 | 29.5303 | 16.1068 | 20194.25 |
| 128 | 25600 | 18340279.75 | 5495975.66 | 18799540.71 | 618.52 | 3437.92 | -1580.45 | 80.9037 | 61.2416 | 44.5667 | 20485.46| 24840114.34 | -6579362.37 | 6759332.80 | 1356.15 | 2015.11 | -3015.84 | 80.9037 | 29.7247 | 14.7617 | 20194.25 |
| 129 | 25800 | 18456400.44 | 6181197.68 | 18475738.97 | 542.61 | 3413.83 | -1657.34 | 80.9183 | 62.2404 | 43.5948 | 20486.92| 25100750.82 | -6173600.71 | 6153377.34 | 1250.03 | 2042.21 | -3043.28 | 80.9183 | 29.9065 | 13.4121 | 20194.27 |
| 130 | 26000 | 18557296.85 | 6861320.99 | 18136696.97 | 466.29 | 3386.94 | -1732.84 | 80.9329 | 63.1799 | 42.5948 | 20488.25| 25340055.70 | -5762592.48 | 5542192.50 | 1142.85 | 2067.58 | -3068.13 | 80.9329 | 30.0769 | 12.0586 | 20194.33 |
| 131 | 26200 | 18642891.84 | 7535786.84 | 17782700.34 | 389.61 | 3357.26 | -1806.88 | 80.9474 | 64.0625 | 41.5686 | 20489.46| 25557826.37 | -5346687.15 | 4926297.85 | 1034.70 | 2091.18 | -3090.38 | 80.9474 | 30.2372 | 10.7016 | 20194.40 |
| 132 | 26400 | 18713120.59 | 8204041.42 | 17414046.45 | 312.64 | 3324.83 | -1879.40 | 80.9620 | 64.8906 | 40.5180 | 20490.55| 25753878.51 | -4926238.30 | 4306216.92 | 925.68 | 2113.01 | -3109.99 | 80.9620 | 30.3886 | 9.3415 | 20194.50 |
| 133 | 26600 | 18767930.54 | 8865536.24 | 17031044.12 | 235.43 | 3289.67 | -1950.35 | 80.9766 | 65.6668 | 39.4444 | 20491.51| 25928046.24 | -4501603.38 | 3682476.78 | 815.87 | 2133.04 | -3126.97 | 80.9766 | 30.5324 | 7.9788 | 20194.62 |
| 134 | 26800 | 18807281.45 | 9519728.61 | 16634013.40 | 158.06 | 3251.81 | -2019.68 | 80.9912 | 66.3935 | 38.3496 | 20492.35| 26080182.28 | -4073143.33 | 3055607.54 | 705.38 | 2151.26 | -3141.28 | 80.9912 | 30.6696 | 6.6138 | 20194.75 |
| 135 | 27000 | 18831145.44 | 10166081.94 | 16223285.32 | 80.57 | 3211.28 | -2087.32 | 81.0058 | 67.0732 | 37.2349 | 20493.05| 26210158.02 | -3641222.33 | 2426141.95 | 594.29 | 2167.65 | -3152.93 | 81.0058 | 30.8014 | 5.2469 | 20194.91 |
| 136 | 27200 | 18839506.92 | 10804066.20 | 15799201.61 | 3.04 | 3168.12 | -2153.22 | 81.0204 | 67.7083 | 36.1017 | 20493.61| 26317863.70 | -3206207.48 | 1794614.94 | 482.69 | 2182.19 | -3161.89 | 81.0204 | 30.9291 | 3.8787 | 20195.07 |
| 137 | 27400 | 18832362.68 | 11433158.26 | 15362114.47 | -74.48 | 3122.37 | -2217.34 | 81.0350 | 68.3014 | 34.9513 | 20494.03| 26403208.41 | -2768468.48 | 1161563.12 | 370.69 | 2194.89 | -3168.17 | 81.0350 | 31.0536 | 2.5093 | 20195.26 |
| 138 | 27600 | 18809721.82 | 12052842.32 | 14912386.26 | -151.92 | 3074.05 | -2279.63 | 81.0495 | 68.8546 | 33.7848 | 20494.29| 26466120.27 | -2328377.32 | 527524.41 | 258.38 | 2205.71 | -3171.76 | 81.0495 | 31.1760 | 1.1393 | 20195.45 |
| 139 | 27800 | 18771605.74 | 12662610.22 | 14450389.26 | -229.22 | 3023.21 | -2340.02 | 81.0641 | 69.3702 | 32.6034 | 20494.41| 26506546.38 | -1886307.95 | -106962.48 | 145.85 | 2214.67 | -3172.66 | 81.0641 | 31.2976 | -0.2310 | 20195.66 |
| 140 | 28000 | 18718048.14 | 13261961.87 | 13976505.36 | -306.32 | 2969.89 | -2398.49 | 81.0787 | 69.8505 | 31.4080 | 20494.36| 26524452.96 | -1442636.00 | -741358.51 | 33.20 | 2221.74 | -3170.85 | 81.0787 | 31.4193 | -1.6012 | 20195.88 |
| 141 | 28200 | 18649094.96 | 13850405.58 | 13491125.79 | -383.16 | 2914.14 | -2454.97 | 81.0933 | 70.2976 | 30.1997 | 20494.14| 26519825.30 | -997738.44 | -1375124.71 | -79.47 | 2226.92 | -3166.36 | 81.0933 | 31.5423 | -2.9710 | 20196.11 |
| 142 | 28400 | 18564804.36 | 14427458.43 | 12994650.83 | -459.68 | 2855.99 | -2509.44 | 81.1079 | 70.7135 | 28.9793 | 20493.75| 26492667.83 | -551993.25 | -2007722.68 | -192.08 | 2230.21 | -3159.17 | 81.1079 | 31.6677 | -4.3399 | 20196.36 |
| 143 | 28600 | 18465246.67 | 14992646.60 | 12487489.47 | -535.82 | 2795.50 | -2561.83 | 81.1225 | 71.1001 | 27.7477 | 20493.19| 26443004.08 | -105779.13 | -2638615.03 | -304.52 | 2231.61 | -3149.30 | 81.1225 | 31.7965 | -5.7076 | 20196.61 |
| 144 | 28800 | 18350504.31 | 15545505.76 | 11970059.19 | -611.52 | 2732.71 | -2612.12 | 81.1370 | 71.4594 | 26.5057 | 20492.44| 26370876.64 | 340524.86 | -3267265.84 | -416.70 | 2231.11 | -3136.76 | 81.1370 | 31.9299 | -7.0736 | 20196.89 |
| 145 | 29000 | 18220671.77 | 16085581.37 | 11442785.56 | -686.72 | 2667.67 | -2660.26 | 81.1516 | 71.7932 | 25.2539 | 20491.50| 26276347.20 | 786539.57 | -3893141.12 | -528.53 | 2228.72 | -3121.55 | 81.1516 | 32.0690 | -8.4377 | 20197.17 |
| 146 | 29200 | 18075855.49 | 16612429.03 | 10906101.98 | -761.35 | 2600.44 | -2706.21 | 81.1662 | 72.1031 | 23.9932 | 20490.37| 26159496.40 | 1231886.12 | -4515709.24 | -639.90 | 2224.43 | -3103.69 | 81.1662 | 32.2150 | -9.7993 | 20197.48 |
| 147 | 29400 | 17916173.85 | 17125614.83 | 10360449.36 | -835.36 | 2531.06 | -2749.94 | 81.1808 | 72.3908 | 22.7240 | 20489.05| 26020423.82 | 1676186.23 | -5134441.39 | -750.73 | 2218.25 | -3083.19 | 81.1808 | 32.3690 | -11.1581 | 20197.80 |
| 148 | 29600 | 17741757.01 | 17624715.61 | 9806275.76 | -908.69 | 2459.60 | -2791.41 | 81.1954 | 72.6580 | 21.4472 | 20487.53| 25859247.91 | 2119062.48 | -5748812.03 | -860.92 | 2210.19 | -3060.08 | 81.1954 | 32.5323 | -12.5136 | 20198.14 |
| 149 | 29800 | 17552746.86 | 18109319.35 | 9244036.12 | -981.28 | 2386.10 | -2830.60 | 81.2100 | 72.9061 | 20.1632 | 20485.80| 25676105.81 | 2560138.69 | -6358299.33 | -970.37 | 2200.26 | -3034.36 | 81.2100 | 32.7061 | -13.8654 | 20198.50 |
| 150 | 30000 | 17349296.93 | 18579025.42 | 8674191.88 | -1053.08 | 2310.63 | -2867.46 | 81.2245 | 73.1367 | 18.8726 | 20483.88| 25471153.32 | 2999040.21 | -6962385.62 | -1079.01 | 2188.45 | -3006.07 | 81.2245 | 32.8916 | -15.2130 | 20198.89 |
| 151 | 30200 | 17131572.24 | 19033444.92 | 8097210.64 | -1124.02 | 2233.25 | -2901.96 | 81.2391 | 73.3511 | 17.5758 | 20481.74| 25244564.72 | 3435394.24 | -7560557.81 | -1186.72 | 2174.78 | -2975.23 | 81.2391 | 33.0902 | -16.5560 | 20199.30 |
| 152 | 30400 | 16899749.24 | 19472200.95 | 7513565.87 | -1194.05 | 2154.01 | -2934.09 | 81.2537 | 73.5507 | 16.2735 | 20479.40| 24996532.64 | 3868830.14 | -8152307.84 | -1293.43 | 2159.27 | -2941.85 | 81.2537 | 33.3033 | -17.8939 | 20199.75 |
| 153 | 30600 | 16654015.65 | 19894928.92 | 6923736.52 | -1263.12 | 2072.98 | -2963.80 | 81.2683 | 73.7369 | 14.9660 | 20476.84| 24727267.87 | 4298979.76 | -8737133.09 | -1399.03 | 2141.92 | -2905.98 | 81.2683 | 33.5321 | -19.2260 | 20200.22 |
| 154 | 30800 | 16394570.33 | 20301276.80 | 6328206.67 | -1331.16 | 1990.22 | -2991.09 | 81.2829 | 73.9108 | 13.6538 | 20474.08| 24436999.22 | 4725477.73 | -9314536.83 | -1503.45 | 2122.76 | -2867.64 | 81.2829 | 33.7783 | -20.5520 | 20200.74 |
| 155 | 31000 | 16121623.17 | 20690905.42 | 5727465.22 | -1398.13 | 1905.80 | -3015.92 | 81.2975 | 74.0738 | 12.3374 | 20471.10| 24125973.30 | 5147961.79 | -9884028.63 | -1606.59 | 2101.79 | -2826.87 | 81.2975 | 34.0433 | -21.8712 | 20201.29 |
| 156 | 31200 | 15835394.93 | 21063488.74 | 5122005.47 | -1463.96 | 1819.78 | -3038.27 | 81.3121 | 74.2271 | 11.0171 | 20467.91| 23794454.32 | 5566073.07 | -10445124.74 | -1708.37 | 2079.03 | -2783.69 | 81.3121 | 34.3287 | -23.1830 | 20201.89 |
| 157 | 31400 | 15536117.10 | 21418714.07 | 4512324.82 | -1528.61 | 1732.23 | -3058.12 | 81.3266 | 74.3717 | 9.6933 | 20464.51| 23442723.88 | 5979456.43 | -10997348.56 | -1808.69 | 2054.51 | -2738.15 | 81.3266 | 34.6362 | -24.4867 | 20202.54 |
| 158 | 31600 | 15224031.72 | 21756282.37 | 3898924.35 | -1592.03 | 1643.22 | -3075.46 | 81.3412 | 74.5089 | 8.3665 | 20460.90| 23071080.69 | 6387760.70 | -11540231.00 | -1907.48 | 2028.24 | -2690.29 | 81.3412 | 34.9674 | -25.7817 | 20203.24 |
| 159 | 31800 | 14899391.25 | 22075908.44 | 3282308.52 | -1654.16 | 1552.82 | -3090.27 | 81.3558 | 74.6398 | 7.0370 | 20457.08| 22679840.37 | 6790639.06 | -12073310.88 | -2004.65 | 2000.25 | -2640.13 | 81.3558 | 35.3242 | -27.0672 | 20204.00 |
| 160 | 32000 | 14562458.36 | 22377321.22 | 2662984.72 | -1714.95 | 1461.10 | -3102.54 | 81.3704 | 74.7654 | 5.7051 | 20453.05| 22269335.15 | 7187749.26 | -12596135.36 | -2100.11 | 1970.57 | -2587.74 | 81.3704 | 35.7084 | -28.3425 | 20204.82 |
| 161 | 32200 | 14213505.77 | 22660263.95 | 2041462.95 | -1774.35 | 1368.13 | -3112.25 | 81.3850 | 74.8869 | 4.3713 | 20448.82| 21839913.59 | 7578753.93 | -13108260.24 | -2193.80 | 1939.20 | -2533.15 | 81.3850 | 36.1221 | -29.6068 | 20205.72 |
| 162 | 32400 | 13852816.07 | 22924494.43 | 1418255.44 | -1832.31 | 1273.99 | -3119.40 | 81.3996 | 75.0052 | 3.0358 | 20444.38| 21391940.32 | 7963320.90 | -13609250.44 | -2285.62 | 1906.19 | -2476.40 | 81.3996 | 36.5672 | -30.8591 | 20206.68 |
| 163 | 32600 | 13480681.53 | 23169785.21 | 793876.24 | -1888.79 | 1178.74 | -3123.97 | 81.4141 | 75.1216 | 1.6991 | 20439.75| 20925795.67 | 8341123.43 | -14098680.26 | -2375.50 | 1871.56 | -2417.55 | 81.4141 | 37.0458 | -32.0985 | 20207.73 |
| 164 | 32800 | 13097403.87 | 23395923.80 | 168840.88 | -1943.73 | 1082.48 | -3125.96 | 81.4287 | 75.2370 | 0.3614 | 20434.92| 20441875.44 | 8711840.53 | -14576133.84 | -2463.36 | 1835.34 | -2356.65 | 81.4287 | 37.5603 | -33.3241 | 20208.85 |
| 165 | 33000 | 12703294.07 | 23602712.85 | -456334.09 | -1997.10 | 985.26 | -3125.36 | 81.4433 | 75.3524 | -0.9769 | 20429.89| 19940590.45 | 9075157.18 | -15041205.43 | -2549.13 | 1797.56 | -2293.74 | 81.4433 | 38.1129 | -34.5347 | 20210.08 |
| 166 | 33200 | 12298672.17 | 23789970.35 | -1081131.38 | -2048.85 | 887.17 | -3122.18 | 81.4579 | 75.4690 | -2.3154 | 20424.68| 19422366.31 | 9430764.65 | -15493499.80 | -2632.74 | 1758.26 | -2228.88 | 81.4579 | 38.7060 | -35.7292 | 20211.39 |
| 167 | 33400 | 11883867.03 | 23957529.80 | -1705033.44 | -2098.92 | 788.29 | -3116.41 | 81.4725 | 75.5877 | -3.6539 | 20419.28| 18887642.97 | 9778360.74 | -15932632.53 | -2714.11 | 1717.46 | -2162.13 | 81.4725 | 39.3421 | -36.9064 | 20212.81 |
| 168 | 33600 | 11459216.10 | 24105240.33 | -2327522.82 | -2147.30 | 688.70 | -3108.05 | 81.4871 | 75.7097 | -4.9918 | 20413.70| 18336874.42 | 10117650.02 | -16358230.34 | -2793.18 | 1675.20 | -2093.54 | 81.4871 | 40.0237 | -38.0650 | 20214.33 |
| 169 | 33800 | 11025065.20 | 24232966.93 | -2948082.57 | -2193.92 | 588.47 | -3097.11 | 81.5016 | 75.8360 | -6.3290 | 20407.94| 17770528.23 | 10448344.11 | -16769931.44 | -2869.88 | 1631.51 | -2023.18 | 81.5016 | 40.7535 | -39.2035 | 20215.96 |
| 170 | 34000 | 10581768.26 | 24340590.54 | -3566196.62 | -2238.75 | 487.68 | -3083.60 | 81.5162 | 75.9677 | -7.6650 | 20402.02| 17189085.26 | 10770161.90 | -17167385.78 | -2944.14 | 1586.44 | -1951.09 | 81.5162 | 41.5341 | -40.3204 | 20217.71 |
| 171 | 34200 | 10129687.08 | 24428008.19 | -4181350.18 | -2281.75 | 386.42 | -3067.51 | 81.5308 | 76.1059 | -8.9994 | 20395.94| 16593039.13 | 11082829.80 | -17550255.40 | -3015.90 | 1540.02 | -1877.34 | 81.5308 | 42.3682 | -41.4142 | 20219.57 |
| 172 | 34400 | 9669191.07 | 24495133.14 | -4793030.18 | -2322.89 | 284.77 | -3048.86 | 81.5454 | 76.2517 | -10.3319 | 20389.70| 15982895.91 | 11386081.95 | -17918214.70 | -3085.10 | 1492.29 | -1801.99 | 81.5454 | 43.2586 | -42.4831 | 20221.55 |
| 173 | 34600 | 9200656.98 | 24541894.98 | -5400725.61 | -2362.13 | 182.80 | -3027.67 | 81.5600 | 76.4064 | -11.6621 | 20383.31| 15359173.64 | 11679660.50 | -18270950.70 | -3151.68 | 1443.29 | -1725.12 | 81.5600 | 44.2079 | -43.5252 | 20223.65 |
| 174 | 34800 | 8724468.67 | 24568239.74 | -6003927.97 | -2399.43 | 80.61 | -3003.94 | 81.5746 | 76.5711 | -12.9896 | 20376.78| 14722401.91 | 11963315.78 | -18608163.32 | -3215.59 | 1393.06 | -1646.77 | 81.5746 | 45.2187 | -44.5386 | 20225.86 |
| 175 | 35000 | 8241016.76 | 24574129.96 | -6602131.63 | -2434.76 | -21.73 | -2977.68 | 81.5892 | 76.7469 | -14.3138 | 20370.12| 14073121.38 | 12236806.50 | -18929565.64 | -3276.76 | 1341.65 | -1567.02 | 81.5892 | 46.2935 | -45.5213 | 20228.18 |
| 176 | 35200 | 7750698.43 | 24559544.81 | -7194834.27 | -2468.09 | -124.13 | -2948.93 | 81.6037 | 76.9353 | -15.6345 | 20363.33| 13411883.40 | 12499900.00 | -19234884.14 | -3335.15 | 1289.10 | -1485.94 | 81.6037 | 47.4346 | -46.4711 | 20230.60 |
| 177 | 35400 | 7253917.05 | 24524480.12 | -7781537.28 | -2499.38 | -226.51 | -2917.69 | 81.6183 | 77.1374 | -16.9511 | 20356.42| 12739249.45 | 12752372.45 | -19523858.91 | -3390.71 | 1235.45 | -1403.60 | 81.6183 | 48.6443 | -47.3856 | 20233.12 |
| 178 | 35600 | 6751081.95 | 24468948.47 | -8361746.11 | -2528.62 | -328.78 | -2883.99 | 81.6329 | 77.3547 | -18.2632 | 20349.41| 12055790.74 | 12994008.98 | -19796243.92 | -3443.39 | 1180.75 | -1320.06 | 81.6329 | 49.9241 | -48.2624 | 20235.71 |
| 179 | 35800 | 6242608.08 | 24392979.18 | -8934970.74 | -2555.77 | -430.87 | -2847.85 | 81.6475 | 77.5886 | -19.5701 | 20342.29| 11362087.69 | 13224603.92 | -20051807.17 | -3493.15 | 1125.04 | -1235.39 | 81.6475 | 51.2756 | -49.0991 | 20238.36 |
| 180 | 36000 | 5728915.72 | 24296618.43 | -9500726.05 | -2580.80 | -532.68 | -2809.30 | 81.6621 | 77.8404 | -20.8715 | 20335.09| 10658729.46 | 13443960.96 | -20290330.95 | -3539.94 | 1068.38 | -1149.68 | 81.6621 | 52.6996 | -49.8930 | 20241.05 |
| 181 | 36200 | 5210430.16 | 24179929.21 | -10058532.23 | -2603.70 | -634.14 | -2768.37 | 81.6767 | 78.1118 | -22.1666 | 20327.81| 9946313.42 | 13651893.30 | -20511611.98 | -3583.72 | 1010.80 | -1062.98 | 81.6767 | 54.1965 | -50.6415 | 20243.75 |
| 182 | 36400 | 4687581.37 | 24042991.33 | -10607915.17 | -2624.43 | -735.16 | -2725.07 | 81.6912 | 78.4043 | -23.4549 | 20320.46| 9225444.70 | 13848223.83 | -20715461.60 | -3624.46 | 952.37 | -975.37 | 81.6912 | 55.7658 | -51.3419 | 20246.42 |
| 183 | 36600 | 4160803.67 | 23885901.47 | -11148406.85 | -2642.98 | -835.65 | -2679.46 | 81.7058 | 78.7196 | -24.7358 | 20313.05| 8496735.63 | 14032785.28 | -20901705.92 | -3662.12 | 893.12 | -886.94 | 81.7058 | 57.4065 | -51.9913 | 20249.04 |
| 184 | 36800 | 3630535.43 | 23708773.12 | -11679545.77 | -2659.33 | -935.53 | -2631.55 | 81.7204 | 79.0594 | -26.0085 | 20305.59| 7760805.24 | 14205420.35 | -21070186.02 | -3696.67 | 833.11 | -797.74 | 81.7204 | 59.1165 | -52.5872 | 20251.55 |
| 185 | 37000 | 3097218.70 | 23511736.55 | -12200877.31 | -2673.46 | -1034.72 | -2581.39 | 81.7350 | 79.4254 | -27.2723 | 20298.10| 7018278.74 | 14365981.85 | -21220758.00 | -3728.07 | 772.39 | -707.87 | 81.7350 | 60.8927 | -53.1268 | 20253.91 |
| 186 | 37200 | 2561298.88 | 23294938.79 | -12711954.15 | -2685.36 | -1133.13 | -2529.01 | 81.7496 | 79.8197 | -28.5265 | 20290.59| 6269786.99 | 14514332.85 | -21353293.19 | -3756.31 | 711.01 | -617.39 | 81.7496 | 62.7312 | -53.6076 | 20256.09 |
| 187 | 37400 | 2023224.38 | 23058543.57 | -13212336.66 | -2695.01 | -1230.68 | -2474.45 | 81.7642 | 80.2441 | -29.7702 | 20283.06| 5515965.95 | 14650346.74 | -21467678.20 | -3781.36 | 649.03 | -526.38 | 81.7642 | 64.6269 | -54.0272 | 20258.02 |
| 188 | 37600 | 1483446.22 | 22802731.21 | -13701593.23 | -2702.40 | -1327.28 | -2417.76 | 81.7787 | 80.7008 | -31.0024 | 20275.53| 4757456.19 | 14773907.40 | -21563815.04 | -3803.20 | 586.49 | -434.92 | 81.7787 | 66.5735 | -54.3834 | 20259.67 |
| 189 | 37800 | 942417.76 | 22527698.61 | -14179300.75 | -2707.51 | -1422.86 | -2358.97 | 81.7933 | 81.1919 | -32.2223 | 20268.02| 3994902.28 | 14884909.27 | -21641621.23 | -3821.80 | 523.45 | -343.09 | 81.7933 | 68.5640 | -54.6741 | 20260.99 |
| 190 | 38000 | 400594.23 | 22233659.08 | -14645044.91 | -2710.34 | -1517.34 | -2298.13 | 81.8079 | 81.7195 | -33.4288 | 20260.53| 3228952.30 | 14983257.45 | -21701029.83 | -3837.16 | 459.96 | -250.96 | 81.8079 | 70.5903 | -54.8979 | 20261.94 |
| 191 | 38200 | -141567.54 | 21920842.28 | -15098420.62 | -2710.89 | -1610.62 | -2235.29 | 81.8225 | 82.2862 | -34.6208 | 20253.08| 2460257.27 | 15068867.76 | -21741989.52 | -3849.25 | 396.08 | -158.61 | 81.8225 | 72.6434 | -55.0533 | 20262.49 |
| 192 | 38400 | -683609.56 | 21589494.06 | -15539032.34 | -2709.15 | -1702.64 | -2170.50 | 81.8371 | 82.8941 | -35.7972 | 20245.68| 1689470.59 | 15141666.86 | -21764464.66 | -3858.07 | 331.86 | -66.12 | 81.8371 | 74.7139 | -55.1394 | 20262.62 |
| 193 | 38600 | -1225073.01 | 21239876.35 | -15966494.51 | -2705.11 | -1793.31 | -2103.81 | 81.8517 | 83.5460 | -36.9566 | 20238.34| 917247.49 | 15201592.27 | -21768435.29 | -3863.61 | 267.35 | 26.42 | 81.8517 | 76.7919 | -55.1556 | 20262.31 |
| 194 | 38800 | -1765498.66 | 20872266.97 | -16380431.84 | -2698.77 | -1882.54 | -2035.26 | 81.8663 | 84.2442 | -38.0977 | 20231.07| 144244.51 | 15248592.43 | -21753897.19 | -3865.87 | 202.62 | 118.95 | 81.8663 | 78.8673 | -55.1020 | 20261.56 |
| 195 | 39000 | -2304427.22 | 20486959.52 | -16780479.72 | -2690.13 | -1970.27 | -1964.92 | 81.8808 | 84.9915 | -39.2190 | 20223.89| -628881.12 | 15282626.78 | -21720861.84 | -3864.84 | 137.70 | 211.38 | 81.8808 | 80.9301 | -54.9788 | 20260.39 |
| 196 | 39200 | -2841399.75 | 20084263.11 | -17166284.54 | -2679.21 | -2056.42 | -1892.84 | 81.8954 | 85.7905 | -40.3190 | 20216.81| -1401471.99 | 15303665.76 | -21669356.45 | -3860.52 | 72.67 | 303.64 | 81.8954 | 82.9705 | -54.7867 | 20258.80 |
| 197 | 39400 | -3375958.01 | 19664502.26 | -17537504.05 | -2665.99 | -2140.91 | -1819.08 | 81.9100 | 86.6439 | -41.3960 | 20209.83| -2172871.07 | 15311690.86 | -21599423.94 | -3852.92 | 7.57 | 395.64 | 81.9100 | 84.9793 | -54.5268 | 20256.83 |
| 198 | 39600 | -3907644.91 | 19228016.61 | -17893807.68 | -2650.50 | -2223.66 | -1743.69 | 81.9246 | 87.5544 | -42.4482 | 20202.96| -2942422.29 | 15306694.60 | -21511122.88 | -3842.04 | -57.53 | 487.31 | 81.9246 | 86.9481 | -54.2007 | 20254.51 |
| 199 | 39800 | -4436004.85 | 18775160.73 | -18234876.88 | -2632.73 | -2304.60 | -1666.74 | 81.9392 | 88.5247 | -43.4737 | 20196.22| -3709471.10 | 15288680.60 | -21404527.45 | -3827.90 | -122.59 | 578.57 | 81.9392 | 88.8693 | -53.8102 | 20251.89 |
| 200 | 40000 | -4960584.15 | 18306303.88 | -18560405.44 | -2612.69 | -2383.65 | -1588.30 | 81.9538 | 89.5573 | -44.4706 | 20189.59| -4473364.98 | 15257663.53 | -21279727.41 | -3810.50 | -187.55 | 669.34 | 81.9538 | 90.7361 | -53.3573 | 20249.01 |
| 201 | 40200 | -5480931.43 | 17821829.72 | -18870099.80 | -2590.41 | -2460.76 | -1508.41 | 81.9683 | 90.6548 | -45.4366 | 20183.10| -5233454.05 | 15213669.13 | -21136827.98 | -3789.85 | -252.36 | 759.55 | 81.9683 | 92.5431 | -52.8444 | 20245.93 |
| 202 | 40400 | -5996598.01 | 17322136.08 | -19163679.37 | -2565.89 | -2535.84 | -1427.16 | 81.9829 | 91.8193 | -46.3695 | 20176.73| -5989091.61 | 15156734.14 | -20975949.81 | -3765.99 | -316.95 | 849.12 | 81.9829 | 94.2855 | -52.2740 | 20242.68 |
| 203 | 40600 | -6507138.32 | 16807634.65 | -19440876.79 | -2539.15 | -2608.82 | -1344.60 | 81.9975 | 93.0529 | -47.2669 | 20170.48| -6739634.64 | 15086906.36 | -20797228.83 | -3738.91 | -381.28 | 937.97 | 81.9975 | 95.9601 | -51.6486 | 20239.32 |
| 204 | 40800 | -7012110.27 | 16278750.70 | -19701438.24 | -2510.21 | -2679.65 | -1260.81 | 82.0121 | 94.3572 | -48.1263 | 20164.34| -7484444.42 | 15004244.50 | -20600816.14 | -3708.66 | -445.28 | 1026.02 | 82.0121 | 97.5642 | -50.9710 | 20235.88 |
| 205 | 41000 | -7511075.69 | 15735922.71 | -19945123.74 | -2479.09 | -2748.25 | -1175.86 | 82.0267 | 95.7336 | -48.9452 | 20158.31| -8222887.04 | 14908818.24 | -20386877.94 | -3675.25 | -508.91 | 1113.21 | 82.0267 | 99.0963 | -50.2438 | 20232.41 |
| 206 | 41200 | -8003600.71 | 15179602.12 | -20171707.37 | -2445.81 | -2814.57 | -1089.80 | 82.0413 | 97.1828 | -49.7208 | 20152.36| -8954333.91 | 14800708.10 | -20155595.32 | -3638.70 | -572.11 | 1199.45 | 82.0413 | 100.5556 | -49.4699 | 20228.95 |
| 207 | 41400 | -8489256.14 | 14610252.92 | -20380977.53 | -2410.39 | -2878.53 | -1002.73 | 82.0558 | 98.7049 | -50.4504 | 20146.47| -9678162.36 | 14680005.40 | -19907164.17 | -3599.07 | -634.83 | 1284.68 | 82.0558 | 101.9422 | -48.6519 | 20225.52 |
| 208 | 41600 | -8967617.92 | 14028351.32 | -20572737.23 | -2372.87 | -2940.08 | -914.71 | 82.0704 | 100.2994 | -51.1312 | 20140.63| -10393756.12 | 14546812.19 | -19641794.96 | -3556.36 | -697.01 | 1368.82 | 82.0704 | 103.2567 | -47.7924 | 20222.15 |
| 209 | 41800 | -9438267.44 | 13434385.36 | -20746804.28 | -2333.28 | -2999.16 | -825.82 | 82.0850 | 101.9649 | -51.7604 | 20134.79| -11100505.87 | 14401241.14 | -19359712.61 | -3510.63 | -758.60 | 1451.80 | 82.0850 | 104.5002 | -46.8939 | 20218.85 |
| 210 | 42000 | -9900792.01 | 12828854.55 | -20903011.50 | -2291.63 | -3055.72 | -736.13 | 82.0996 | 103.6990 | -52.3352 | 20128.94| -11797809.72 | 14243415.49 | -19061156.29 | -3461.91 | -819.55 | 1533.55 | 82.0996 | 105.6744 | -45.9591 | 20215.66 |
| 211 | 42200 | -10354785.19 | 12212269.45 | -21041206.97 | -2247.97 | -3109.70 | -645.71 | 82.1142 | 105.4985 | -52.8529 | 20123.02| -12485073.81 | 14073468.89 | -18746379.21 | -3410.24 | -879.80 | 1614.00 | 82.1142 | 106.7812 | -44.9901 | 20212.57 |
| 212 | 42400 | -10799847.22 | 11585151.26 | -21161254.22 | -2202.32 | -3161.04 | -554.66 | 82.1288 | 107.3590 | -53.3110 | 20117.00| -13161712.72 | 13891545.36 | -18415648.39 | -3355.67 | -939.31 | 1693.08 | 82.1288 | 107.8229 | -43.9893 | 20209.60 |
| 213 | 42600 | -11235585.38 | 10948031.36 | -21263032.35 | -2154.74 | -3209.71 | -463.04 | 82.1434 | 109.2753 | -53.7069 | 20110.85| -13827150.03 | 13697799.08 | -18069244.49 | -3298.23 | -998.02 | 1770.72 | 82.1434 | 108.8019 | -42.9587 | 20206.76 |
| 214 | 42800 | -11661614.39 | 10301450.95 | -21346436.28 | -2105.24 | -3255.64 | -370.93 | 82.1579 | 111.2408 | -54.0384 | 20104.53| -14480818.79 | 13492394.36 | -17707461.53 | -3237.99 | -1055.88 | 1846.86 | 82.1579 | 109.7207 | -41.9004 | 20204.05 |
| 215 | 43000 | -12077556.77 | 9645960.51 | -21411376.86 | -2053.88 | -3298.80 | -278.42 | 82.1725 | 113.2481 | -54.3037 | 20097.99| -15122162.01 | 13275505.41 | -17330606.63 | -3174.99 | -1112.85 | 1921.43 | 82.1725 | 110.5820 | -40.8163 | 20201.47 |
| 216 | 43200 | -12483043.23 | 8982119.37 | -21457781.01 | -2000.69 | -3339.14 | -185.58 | 82.1871 | 115.2890 | -54.5010 | 20091.22| -15750633.15 | 13047316.27 | -16938999.81 | -3109.28 | -1168.88 | 1994.36 | 82.1871 | 111.3885 | -39.7082 | 20199.02 |

## 2. Ground Site Visibility (50 points)

Satellite communication and commanding is usually performed via a radio link between a ground site and the satellite. A very common computation is therefore to determine the times at which the satellite is in view of the ground station. A satellite is “visible” to the ground site when elevation > 0°.

Using the orbit for Vehicle 1:

a. Find the start and stop times for elevation > 0°; i.e., when this satellite is visible to the POGO ground station.  
b. Find the azimuth at the start and stop of the pass  
c. Find the maximum elevation for the pass.  
d. Find the azimuth at maximum elevation.

Report your results in a table like the following:

| Event        | Time (hh:mm:ss) | Elevation (deg) | Azimuth (deg) |
|--------------|-----------------|-----------------|---------------|
| Rise         |                 |                 |               |
| Max Elevation|                 |                 |               |
| Set          |                 |                 |               |

**Scoring:** Each of the 9 elements of the table will have equal worth (effectively, 5⅝ points), so that a totally correct table will earn the full 50 points.

## 3. Radio Frequency Interference (100 points)

Radio Frequency Interference (RFI) occurs when signals transmitted by a ground station to one satellite, are also received by, and interfere with, the communications system of a second satellite.

An RFI event is most likely to occur when the angular separation between two satellites, as seen from the ground site, is less than some tolerance angle, $\theta$. The geometry of the problem is shown in the following figure.

For this problem, you will derive the steps required to compute the angular separation, $\theta$, between the two satellites, as seen from the POGO ground site.

All vectors used in the computation must be expressed in the same reference frame. Beyond that, the choice of reference frame can be made based on computational convenience.

a. Using vector addition, write expressions for $\vec{r}_{1,s}$ and $\vec{r}_{2,s}$, in terms of the ground site and satellite vectors. (10 pts)

b. Using the definition in the figure, write an equation for finding the angular separation, $\theta$, between satellites 1 and 2 (i.e., find the angle between $\vec{r}_{1,s}$ and $\vec{r}_{2,s}$). (10 pts)  
**Hint:** remember the definitions of the vector operations we've used to compute the angle between two vectors.

c. Compute the ECEF position vector for the POGO ground site, $\vec{r}_S$ (10 pts)

d. The RFI computation requires all vectors to be in a common reference frame. Explain which reference frame you will use to perform the $\theta$ computation. (10 pts)

e. Write the equation(s) that you will use to perform the coordinate transformation(s) implied by your answer in part (d). (10 pts)

f. For each time point that Satellite 1 is in view of the POGO ground site, compute the angular separation $\theta$ between satellites. (20 pts)

Using your orbit predictions for Satellites 1 and 2,

g. Estimate the start and stop times for the period where $\theta$ < 3°. (10 pts)

h. Estimate the magnitude of the minimum separation angle between the two satellites. (10 pts)

i. Estimate the time at which the minimum separation occurs. (10 pts)

## Heading Checks

The following represents a full set of values for problems 1, 2, and 3 for a specific time step.

**Step** 64  
**t** 12800 seconds  
**Time** 20-Apr-2012 03:33:20 UTC  

**SV1 State (TOD)**  
x1 = -9130348.20  
y1 = -24153557.33  
z1 = 5222746.67  
x1dot = 2370.11  
y1dot = -263.58  
z1dot = 3092.46  

**SV2 State (TOD)**  
x2 = -16131692.07  
y2 = -11293317.75  
z2 = 17801664.49  
x2dot = 3068.63  
y2dot = -1507.79  
z2dot = 1825.83  

**SV1 Az** 120.972 Deg  
**SV1 El** 4.797 Deg  
**Sep Angle** 48.151 Deg  

The separation angle of 48.151° corresponds to the angle $\theta$ in the figure from Problem 3.